ner_v2/detectors/temporal/date/en/test.yaml

```diff
@@ -0,0 +1,337 @@
+args:
+  today:
+    dd: 15
+    mm: 2
+    yy: 2019
+tests:
+  en:
+    - message: 17/12/19
+      bot_message: null
+      outputs:
+        - dd: 17
+          mm: 12
+          yy: 2019
+          property: null
+          range: null
+          repeat: null
```

Comment:
We should add the text in each output that was detected as date, because our detectors don't guarantee order

Reply:
Done